backend/run.py

```diff
@@ -0,0 +1,21 @@
+from flask import Flask, render_template, jsonify
+from random import *
+import requests
+
+app = Flask(__name__,
+            static_folder = "../frontend/dist/static",
+            template_folder = "../frontend/dist")
```

Comment:
PEP8 violation. May I suggest to set up an automatic PEP8 compliance checking in CI? It may be done in another PR sometime later. Here's an example configuration to be put into `setup.cfg`:

```ini
[pycodestyle]
# E221 multiple spaces before operator
# E241 multiple spaces after ':'
ignore = E221, E241
max-line-length = 120
```

The linter is called `pycodestyle` (available via pip), it is to be invoked from the project root (where the setup file is) like this: `pycodestyle --show-source .`.

For a complete example see https://github.com/UAVCAN/pydsdl.